src/loom-provider.ts

```diff
@@ -571,8 +573,15 @@ export class LoomProvider {
     return this._client.evmUnsubscribeAsync(payload.params[0])
   }
 
+  private _chainIdToNetVersion() {
+    const chainIdHash = soliditySha3(this._client.chainId)
+      .slice(2)
+      .substr(0, 14)
```

Comment:
Why 14?

Reply:
It's the number of digits valid before give the 53 bits integer error

Reply:
Why not convert the whole hash to BN?

Reply:
It will be ok to convert, but when pass the call to web3 it should will be converted to number which will raise the error of 53bits, I've didn't notice that before because the size of the `ChainID` previous tested, getting only 15 digits allow to get huge number without raise error